deploy/crds/managed_v1alpha1_subjectpermissions_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: managed.openshift.com/v1alpha1
+kind: SubjectPermissions
```

Comment:
I think this file and ` deploy/crds/managed_v1alpha1_subjectpermissions_crd.yaml` can be removed since your type is `SubjectPermission`